include/shape3d.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef SHAPE3DH
+    #define SHAPE3DH
+
+    struct Vector3;
+    struct Color;
+
+    /**struct Shape3D;
+     * @brief Shape3D object
+     */ 
+    class shape3D {
```

Comment:
Please capitalize the first letter as such: Shape3D.

And also add this to the indie namespace, it would be weird to have it for some classes and not others

Reply:
Done in this commit "MODIFIED: shape3D class in indie namespace, shape3D method in lower case"